docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #63.            raise ValueError(f"The length of include ({len(include)}) does not match the total number of subsequences ({k})")
</pre><p>I think this should be <code>total number of subsequences ({l})</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Yes. Will be fixed in the next set of commits